main/zoe/guide/price-authority.md

```diff
@@ -46,6 +46,11 @@ This means that the `priceAuthority` asserts that when `timestamp` according to
 `amountIn` and `amountOut` are ERTP amounts for the `brandIn` and `brandOut` you
 requested.
 
+## Mutable price quotes
+
+A `MutableQuote` object is the same as a `PriceQuote`, except that you can cancel it or
+change its trigger levels to specified values without requiring a second `Promise`. 
```

Comment:
A `PriceQuote` is a record (has fields rather than methods), while `MutableQuote` is an object (methods, no fields).  `MutableQuote`'s method `getPromise()` returns a promise for a `PriceQuote`, which is the same Promise returned by `quoteWhenLTE()` and its siblings.